guides/cookbook.md

```diff
@@ -879,6 +1137,71 @@ be used to determine whether an Ion `int` can be fully represented as a `number`
 if not, call `Reader.bigIntValue()` to avoid loss of precision.
 </div>
 
+<div class="tabpane Python" markdown="1">
+Use of separate APIs to read different numeric types is not necessary in Ion-Python. Via both
+simpleion and the event-based API, Ion int will be read as Python `int`, Ion decimal will be
+read as Python `Decimal`, and Ion float will be read as Python `float`. When necessary, these
+values can be differentiated via duck typing.
+
+#### Using simpleion
+
+```python
+from decimal import Decimal
+
+data = u'1.23456 1.2345e6 123456'
```

Comment:
consider adding `12345678901234567890` for consistency with the other impls; yep, also an int

Reply:
Done.